task/buildah-oci-ta/0.3/buildah-oci-ta.yaml

```diff
@@ -725,3 +754,26 @@ spec:
         requests:
           cpu: 100m
           memory: 256Mi
+  sidecars:
+    - name: indy-generic-proxy
+      image: quay.io/factory2/indy-generic-proxy-service:latest-prod
```

Comment:
What is the provenance of this image?

Reply:
I believe we need to follow konflux pattern to build the image if we agree the sidecar proposal.